apps/api/src/app/workflows-v2/e2e/generate-preview.e2e.ts

```diff
@@ -520,150 +520,6 @@ describe('Workflow Step Preview - POST /:workflowId/step/:stepId/preview #novu-v
     });
   });
 
-  it('should transform tip tap node to liquid variables', async () => {
```

Comment:
Same reason as explained here: https://github.com/novuhq/novu/pull/7544#discussion_r1922509629
Its already covered by unit test.

I'm not yet confident removing more than this but I can see us having just few integration e2e tests for preview and having unit tests for each step rendering logic.